spec/support/vcr_cassettes/caseflow/not_found_404_response.yml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
I noticed that this cassette is identical to the existing `caseflow/not_found.yml` cassette. Why was that existing cassette not used?

Reply:
Sure, it's really tough to see with GitHub's formatting, but the new cassette has an updated SSN header value. 

The old cassette for `not_found` is for an ICN that doesn't match the ICN in the token that's sent. So we needed a version of `not_found` with an SSN header matching the ICN in the `mpi/find_candidate/valid response`.

Reply:
Ah, I see! That makes a lot of sense. Thanks for explaining!

Because I was curious, I tested modifying the SSN header in the existing `caseflow/not_found.yml` cassette, and it looks like all the tests that use it are still passing, so I think you could modify it directly without any downsides.

Reply:
Excellent—I pushed up those changes,  thanks! 👍🏻

Reply:
Looks great, thank you!